packages/nimble-components/src/table-column/icon/group-header-view/styles.ts

```diff
@@ -0,0 +1,29 @@
+import { css } from '@microsoft/fast-element';
+import { display } from '@microsoft/fast-foundation';
+import {
+    bodyFont,
+    bodyFontColor,
+    smallPadding
+} from '../../../theme-provider/design-tokens';
+
+export const styles = css`
+    ${display('inline-flex')}
+
+    :host {
+        gap: ${smallPadding};
+        align-items: center;
+    }
+
+    .no-shrink {
```

Comment:
I don't see this being used. Remove?

Reply:
Somewhere along the way, some code got accidentally removed. I updated the templates for the icons and spinners to include the class `no-shrink`. I also updated the templates to put label text inside a `span` that conditionally sets the `title`. Thanks for catching this.